docs/_static/nuts-registry.yaml

```diff
@@ -27,20 +27,12 @@ paths:
                 $ref: '#/components/schemas/Event'
         '400':
           description: "incorrect data"
-  /api/vendor/{id}/claim:
+  /api/organization:
```

Comment:
Can we leave the old endpoint, give it a different operationId, ignore the id and log that it is deprecated.

Reply:
Fixed